lib/Zonemaster/Backend/Config.pm

```diff
@@ -58,22 +72,130 @@ sub BackendDBType {
     }
 }
 
-sub DB_user {
-    my ($self) = @_;
+=head2 MYSQL_database
+
+Returns the L<MYSQL.database|https://github.com/zonemaster/zonemaster-backend/blob/master/docs/Configuration.md#mysql>
```

Comment:
The fragment should be `#mysql-section`, it is even possible to point directly to the right section with `#database`

Reply:
It's quite error prone to link to the auto-suffixed anchors. But let's try it out. If it's too painful we'll go back to just linking to the headings.

Reply:
Ok, we'll see how it goes, to be honest I don't have a strong opinion on this. I just wanted to know if you made this choice on purpose and I now totally understand your first move. Now that you updated it, let's go for a try ;)